### ISSUE TITLE
Breaking Ties

### ISSUE THREAD

**Issue by AfterNetwork** [AfterNetwork/AFNNetworkK12-Creations#15]
I have to break ties with the scratch community. I've done the worst things in the scratch community and I regret it. Scratch has become the worst part of my social life online. I was given chances. but didn't listen. I regret all the actions I have done but it's time to move on from this horrible part. It's affected the way I think, and I don't want to think that way. Anyone who is in scratch just makes me relive this community. I feel sorry. I want to one last goodbye and sorry to the people who I have affected. I'm sorry for this entire situation. Thank you.

Goodbye,
After

**Comment by AlphaZetta** [comment]
It's the worst part of your life because of **you**. You were never gonna last with your personality and it took a permanent ban to stop you. You've apologised like this countless times, and instead of facing the community with consequences, you avoid them. I would like your parent's email, this must go to them for me to be happy.

**Comment by AfterNetwork** [comment]
> 
> 
> It's the worst part of your life because of **you**. You were never gonna last with your personality and it took a permanent ban to stop you. You've apologised like this countless times, and instead of facing the community with consequences, you avoid them. I would like your parent's email, this must go to them for me to be happy.

Just give me a break. The person that mostly led the worst time of my period with Scratch. The currency era. Just leave me be, and I'll be silent.

**Comment by AlphaZetta** [comment]
> > It's the worst part of your life because of **you**. You were never gonna last with your personality and it took a permanent ban to stop you. You've apologised like this countless times, and instead of facing the community with consequences, you avoid them. I would like your parent's email, this must go to them for me to be happy.
> 
> Just give me a break. The person that mostly led the worst time of my period with Scratch. The currency era. Just leave me be, and I'll be silent.

No, I want you to face real-life consequences. I did not lead your time. YOU did. I just ran a service that you made so problematic. And that's a fact.

**Comment by AlphaZetta** [comment]
> Look, just leave me alone. Close this here.

Give me your parent's email and I will :)

**Comment by AfterNetwork** [comment]
> 
> 
> > Look, just leave me alone. Close this here.
> 
> Give me your parent's email and I will :)

I will not because of privacy reasons.

**Comment by AlphaZetta** [comment]
> > > Look, just leave me alone. Close this here.
> > 
> > 
> > Give me your parent's email and I will :)
> 
> I will not because of privacy reasons.

Okay, then tell your parents what has been happening over the past 2 years.

**Comment by AfterNetwork** [comment]
> 
> 
> > > > Look, just leave me alone. Close this here.
> > > 
> > > 
> > > Give me your parent's email and I will :)
> > 
> > 
> > I will not because of privacy reasons.
> 
> Okay, then tell your parents what has been happening over the past 2 years.

just did, my parents told me to think about my actions and thats it

**Comment by AlphaZetta** [comment]
> > > > > Look, just leave me alone. Close this here.
> > > > 
> > > > 
> > > > Give me your parent's email and I will :)
> > > 
> > > 
> > > I will not because of privacy reasons.
> > 
> > 
> > Okay, then tell your parents what has been happening over the past 2 years.
> 
> just did, my parents told me to think about my actions and thats it

Thank you. I will now leave you alone, and I wish you a good life.
